### PR TITLE
Updated Edge Shopping Privacy Whitepaper: added inbox coupons

### PR DESCRIPTION
**Pull request by ihabrizk**
AB#44626679

### CODE REVIEW

**Comment by ihabrizk** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by ihabrizk** [comment]
@microsoft-github-policy-service agree company="Microsoft"

Best,
Ihab - Cairo Incubations PM

P.S. I don't expect you to respond to my emails outside your work hours.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
 
+![Shopping and coupons](./media/inboxcoupons.png)
+
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and is accessible to you. Inbox coupons will not appear when using **Guest mode** or **InPrivate**.
+
+![Shopping and coupons](./media/inboxcoupons-settings.png)
+
+To control which retailers' emails get scanned by Microsoft Edge, complete the following actions:
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:605]:
```suggestion
To control which retailers' emails get scanned by Microsoft Edge:
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
...
+![Shopping and coupons](./media/inboxcoupons-settings.png)
+
+To control which retailers' emails get scanned by Microsoft Edge, complete the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar 
+1.  Go to the Inbox coupons card and click on the “...” button 
+1.  Click on the **“Settings”** link and view retailers in your inbox 
+1.  Control retailer email preferences via toggles 
+
+To turn off Inbox coupons, completing the following actions:
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:612]:
```suggestion
To turn off Inbox coupons:
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
 
+![Shopping and coupons](./media/inboxcoupons.png)
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:599]:
```suggestion
![Coupons for various websites](./media/inboxcoupons.png)
```
unique alt-text

File: microsoft-edge/privacy-whitepaper/index.md [hunk 4]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
...
+
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and is accessible to you. Inbox coupons will not appear when using **Guest mode** or **InPrivate**.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:601]:
```suggestion
To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails that are sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and are accessible to you. Inbox coupons don't appear when using **Guest mode** or **InPrivate**.
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 5]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
...
+
+![Shopping and coupons](./media/inboxcoupons-settings.png)
+
+To control which retailers' emails get scanned by Microsoft Edge, complete the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar 
+1.  Go to the Inbox coupons card and click on the “...” button 
+1.  Click on the **“Settings”** link and view retailers in your inbox 
+1.  Control retailer email preferences via toggles
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:610]:
```suggestion
To control which retailers' emails are scanned by Microsoft Edge:

1.  On the **Shopping** sidebar, click the **Your Shopping** tab.

1.  In the **Inbox coupons** card, click the **Settings** (**...**) button:

    ![Shopping and coupons](./media/inboxcoupons-settings.png)

1.  In the **Retailers in your inbox** section, expand the section for a retailer, and then turn the toggle on or off.
```

File: microsoft-edge/privacy-whitepaper/media/inboxcoupons-settings.png
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/media/inboxcoupons-settings.png:1]: This picture should be removed. You can move the other picture down into its place.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 6]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
...
+1.  Go to the Inbox coupons card and click on the “...” button 
+1.  Click on the **“Settings”** link and view retailers in your inbox 
+1.  Control retailer email preferences via toggles 
+
+To turn off Inbox coupons, completing the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar 
+1.  Go to the Inbox coupons card and click on the “...” button 
+1.  Select the email account you wish to disconnect and click on the **“Settings”** link 
+1.  **“Disconnect email”**
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:617]:
```suggestion
To turn off Inbox coupons:

1.  On the **Shopping** sidebar, click the **Your Shopping** tab.

1.  In the **Inbox coupons** card, click the **Settings** (**...**) button.

1.  Select an email account, and then click the **Settings** link.

1.  Click the **Disconnect email** button.
```

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Finished a Writer/Editor pass.

File: microsoft-edge/privacy-whitepaper/media/inboxcoupons-settings.png
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/media/inboxcoupons-settings.png:1]: @apryzgoda - recommend entering comment on the specific line in the Files Changed view, for tracking and to see my suggested changes.  In my VS Code local copy of the .md I might try your suggestion to see where the remaining .png would go in my suggested steps.

Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/media/inboxcoupons-settings.png:1]: @apryzgoda - i revised my [Suggested change](https://github.com/MicrosoftDocs/edge-developer/pull/2589/files#r1194224979) so as to keep both .png files but moved them into their places within the steps.   If you still want to delete one .png, that's easy, from the perspective of my change that implements some of your comment.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 7]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
 
+![Shopping and coupons](./media/inboxcoupons.png)
+
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:600]:
```suggestion
```
moved png down into steps per @apryzgoda comment

Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:600]:
```suggestion

```
moved png down into steps per @apryzgoda comment

File: microsoft-edge/privacy-whitepaper/index.md [hunk 8]
```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
...
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and is accessible to you. Inbox coupons will not appear when using **Guest mode** or **InPrivate**.
+
+![Shopping and coupons](./media/inboxcoupons-settings.png)
+
+To control which retailers' emails get scanned by Microsoft Edge, complete the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:607]: missing step 1:  how do I get to the shopping sidebar?  I'm checking actual UI now...

File: microsoft-edge/privacy-whitepaper/index.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:610]: i haven't reproduced the actual UI yet; i can't actually follow the draft steps or revised steps.  I'm not logged in, and there is no "Inbox coupons" card.

File: microsoft-edge/privacy-whitepaper/index.md (hunk 6 above)
Comment (ihabrizk) [reply microsoft-edge/privacy-whitepaper/index.md:614]: @mikehoffms the disconnect email is a button (in css) but appears as a link

image

image

File: microsoft-edge/privacy-whitepaper/index.md (hunk 5 above)
Comment (ihabrizk) [reply microsoft-edge/privacy-whitepaper/index.md:607]: @mikehoffms Inbox coupons is 100% in US market and we are flighting in UK/AU by end of May and will start our flights to DE/FR in June. If you are having trouble viewing our experience in the US please let me know.

To access the experience outside the US you will need to use Edge Launcher and run our flag "--enable-features=msShoppingExp52"

image

**Review by ihabrizk — COMMENTED** [review]
@mikehoffms i am not sure if all Edge versions have **Shopping** under the **Settings and more** (**...**) button for the **Shopping** sidebar to appears - my version (113.0.1774.42) doesn't.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 9]
```diff
@@ -595,6 +595,31 @@ When you go to a website, or save an item to your Collections, Microsoft Edge lo
 Microsoft also sends the product price, product image, product name, ratings, and reviews, along with information about Microsoft Edge and your operating system version to the service.  The data is sent over HTTPS with a randomly generated identifier and cookies if cookies are allowed.
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails that are sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and are accessible to you. Inbox coupons don't appear when using **Guest mode** or **InPrivate**.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:598]:
```suggestion

To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails that are sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and are accessible to you. Inbox coupons don't appear when using **Guest mode** or **InPrivate**.
```
add blank line to break up long paragraph - 2nd paragraph, this way, groups the sentences that contain "coupon".

**Review by mikehoffms — APPROVED** [review]
approved per Writer/Editor pass, assuming the blank line is added in order to separate the new paragraph per [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2589/files#r1195633883).

**Comment by mikehoffms** [comment]
@ihabrizk - if you have this UI, recommend you click "Re-request review" next to @apryzgoda in the Reviewers section at top of PR.
And also message apryzgoda asking to review/approve this PR.

offline, I will:
* ask who is required to approve this PR (w/ green checkmark).  
* request merging/publishing this PR.

**Comment by mikehoffms** [comment]
@ihabrizk -- sometimes Shopping doesn't appear in "Settings and more" Edge menu, if you've opened Shopping already.  don't know the details/ exact conditions that cause the command to be present or absent.

I recommend you try again in Edge 113: fully close Shopping, then expect the command to appear.

**Review by apryzgoda — APPROVED** [review]
Approved from Edge Privacy POV given the blank line is added per mikehoffms suggestion.

**Comment by ihabrizk** [comment]
@mikehoffms could you share your Microsoft Team's alias, I would like to connect with you to debug why you cannot see the experience

File: microsoft-edge/privacy-whitepaper/index.md [hunk 10]
```diff
@@ -596,6 +596,32 @@ Microsoft also sends the product price, product image, product name, ratings, an
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
 
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails that are sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and are accessible to you. Inbox coupons don't appear when using **Guest mode** or **InPrivate**.
+
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:600]:
```suggestion

The following steps assume that you have already signed up for coupons.

```
please try to make the wording more specific & technically accurate than my suggested wording.

**Comment by ihabrizk** [comment]
@mikehoffms incorporated your suggestion revising the text to say:

image

**Review by Reezaali — APPROVED** [review]
Privacy approval received over Teams. Approving to proceed with the merge.

**Pull request merged**